services/conversation/v1-exp.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright 2016 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+module.exports = function (RED) {
+  var cfenv = require('cfenv'), watson = require('watson-developer-cloud'), service = null,
+    sUsername = null, sPassword = null;
+  
+  service = cfenv.getAppEnv().getServiceCreds(/conversation/i);
+
+  if (service) {
+    sUsername = service.username;
+    sPassword = service.password;
+  }
+
+  RED.httpAdmin.get('/watson-conversation/vcap', function (req, res) {
+    res.json(service ? {bound_service: true} : null);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'bound_service' is not in camel case.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2775458781/issues/source?bid=3330694&fileBranchId=3363974#l29)